pandas/sparse/tests/test_list.py

```diff
@@ -113,6 +113,13 @@ def test_getitem(self):
                 tm.assert_almost_equal(splist[i], arr[i])
                 tm.assert_almost_equal(splist[-i], arr[-i])
 
+    def test_validate_bool_args(self):
+        invalid_values = [1, "True", [1,2,3], 5.0]
+        lst = SparseList(self.na_data)
+        for value in invalid_values:
```

Comment:
or just leave out this test, since its deprecated it's not that important to test this